pkg/console/subresource/configmap/configmap_test.go

```diff
@@ -68,6 +76,7 @@ func TestDefaultConfigMap(t *testing.T) {
 					Spec:   configv1.ConsoleSpec{},
 					Status: configv1.ConsoleStatus{},
 				},
+				newConfig: &corev1.ConfigMap{},
```

Comment:
Probably we now have a case for more than one test.  

```
tests := []struct {
    name string 
    args args
    want *corev1.ConfigMap
}{ 
   name: "Generate default configmap"
}, {
   name: "Generate configmap when extra branding config from openshift-config-managed is provided"
}, {
   name: "Ben should have written an extra test for operatorConfig.UnsupportedConfigOverrides"
}
```
I shall have to go write the test I forgot to write now :)

Reply:
Great idea.  You should probably wait until this PR merges but I will add another unit test for managedConfigs.

Reply:
Yeah I'll wait on mine to avoid a conflict.